test/remote/gateways/remote_trans_first_test.rb

```diff
@@ -70,12 +70,12 @@ def test_failed_purchase
     assert_equal 'Insufficient funds', response.message
   end
 
-  def test_successful_refund
```

Comment:
@nfarve Here's a past example of what I've done in this kind of situation: https://github.com/activemerchant/active_merchant/blob/9c44975612deec9d6795d1ff385071a9ae5106ae/test/remote/gateways/remote_global_collect_test.rb#L79